twpca/crossval.py

```diff
@@ -11,94 +11,52 @@
 
 __all__ = ['cross_validate', 'hyperparam_search']
 
-def k_fold_iter(N, K):
-    """K-fold cross validation iterator
+def cross_validate(model, data, nfits, censor_prob, **fit_kw):
+    """Runs cross-validation on TWPCA model.
 
-    Args:
-        N: int, number of datapoitns
-        K: int, number of folds
-    """
-    idx = np.random.permutation(N)
-    for k in range(K):
-        train_idx = idx[idx % K != k]
-        test_idx = idx[idx % K == k]
-        yield train_idx, test_idx
-
-def leave_k_out_iter(N, K):
-    """Leave K out cross validation iterator
+    Args
+    ---
+    model : TWPCA model instance
+    data (ndarray) : data tensor, trials x time x neurons
+    nfits (int) : number of cross-validation runs
+    censor_prob (float) : probability of setting element of data to nan
```

Comment:
why censor_prob? could we stick with keep_prob or drop_prob like folks use for dropout?

also make the doc more descriptive, like "fraction of data to hold out for each cross validation run"